Model/Log.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2020 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Model;
+
+use Magento\Framework\Model\AbstractModel;
+
+/**
+ * Class Log
+ * @package Bolt\Boltpay\Model
+ */
+class Log extends AbstractModel implements \Magento\Framework\DataObject\IdentityInterface
+{
+    const CACHE_TAG = 'bolt_log';
+
+    protected $_cacheTag = self::CACHE_TAG;
```

Comment:
Is this used anywhere?

Reply:
This is following the Core Magento. $_cacheTag is a unique identifier that we declare for caching the model. See here - https://screencast.com/t/1whtN6qZx